http/file_server.ts

```diff
@@ -855,21 +856,25 @@ function main() {
     console.log(message);
   }
 
+  // TODO(petamoriken): Migrate `Deno.ServeTcpOptions | (Deno.ServeTcpOptions & Deno.TlsCertifiedKeyOptions)` in v2
```

Comment:
What does this mean?

Reply:
I used `Deno.ServeTlsOptions`, but it was renamed to `Deno.ServeTcpOptions` in Deno v2 (https://github.com/denoland/deno/pull/25369), so I wrote flatly this type 😭